webtask.json

```diff
@@ -9,6 +9,7 @@
   "useHashName": false,
   "docsUrl": "https://github.com/auth0/auth0-deploy-cli/blob/master/README.md",
   "logoUrl": "http://openid.net/wordpress-content/uploads/2016/05/auth0-logo-blue.png",
+  "codeUrl": "https://raw.githubusercontent.com/auth0-extensions/auth0-deploy-cli-extension/master/index.js",
```

Comment:
Hi. Could you please replace `codeUrl` with `https://raw.githubusercontent.com/auth0-extensions/auth0-deploy-cli-extension/master/build/bundle.js`? `./index.js` isn't extension's code.

Reply:
Done @zxan1285